controllers/ibmpowervsmachine_controller.go

```diff
@@ -87,6 +88,7 @@ func (r *IBMPowerVSMachineReconciler) Reconcile(ctx context.Context, req ctrl.Re
 		log.Info("Machine Controller has not yet set OwnerRef")
 		return ctrl.Result{}, nil
 	}
+	log = log.WithValues("machine", machine.Name)
```

Comment:
needs to be ibmpowervsmachine vs machine?

Reply:
will change to ibmPowerVSMachine to avoid confustion